docs/docs/Developing_with_JAC/Language_Features/dataTypes.md

```diff
@@ -4,7 +4,7 @@ title : Data Types
 
 
 ## Data types
-Jaseci is a dynamically typed language so there is no need to specify the data type.
+Jaseci is a dynamically typed language so there is no need to declare the data type.
```

Comment:
JAC is a dynamically typed language so there is no need to declare the data type.